api/src/main/java/com/datatorrent/api/DAG.java

```diff
@@ -281,6 +306,20 @@
   public abstract OperatorMeta getMeta(Operator operator);
 
   /**
+   * Return all operators present in the DAG.
+   * @param <T>
+   * @return
+   */
+  public <T extends OperatorMeta> Collection<T> getAllOperators();
+
+  /**
+   * Returns all Streams present in the DAG.
+   * @param <T>
+   * @return
+   */
+  public <T extends StreamMeta> Collection<T> getAllStreams();
+
+  /**
```

Comment:
In these methods, how about having the string Meta in the method names for retrieving the OperatorMeta. We already have that for existing methods like DAG#getOperatorMeta(..). This would also allow us to accommodate getOperator() type methods in future that return the underlying component.